frontend_app_thread/src/css/index.styl

```diff
@@ -43,6 +43,6 @@
 
 @media (max-width: max-xs)
   .thread .timeline
-    height "calc(100vh - %s)" % (headerHeight * 2 + 25px) // INFO - 2021-08-08 - GB - 25px is the thread__contentpage__top height
+    height "calc(100vh - %s)" % (headerHeight + 25px) // INFO - 2021-08-08 - GB - 25px is the thread__contentpage__top height
```

Comment:
Do we keep the header in mobile?

Reply:
No, I didn't make a lot of tests on mobile yet, but I think the 2 headerHeights were: main header + app header. So I deleted only one.